cmake/MacOSX/fix_dependencies.rb

```
@@ -13,511 +13,504 @@
-        debug "Handle binary #{$bin_dir + content}"
-        handleBinary($bin_dir + content)
+    ## TODO does not really work yet. Would copy all plugin dependencies into plugin_dir
+    ##  but it SHOULD copy them into lib and relink relatively if necessary.
+    ##  Currently we just copy the plugins and hope they have dependencies that are covered
+    ##  by the usual libs. Should work for Qt (our only plugin).
+    if !@plugin_dir.nil?
+      debug "HANDLING PLUGIN DIR"
+      for content in Dir.glob("#{@plugin_dir}/**/*.dylib")
+        debug "Handle dylib #{@plugin_dir + content}"
+        handleDyLib(@plugin_dir + content, File.dirname(@plugin_dir + content), [])
       end
```

RuboCop (1.82.1)

[convention] Prefer `each` over `for`.

(Style/For)